kubernetes/job/resource.yaml

```diff
@@ -57,7 +57,7 @@ subjects:
 apiVersion: batch/v1
 kind: Job
 metadata:
-  name: razeedeploy-job
+  name: "razeedeploy-job{{{ UUID }}}"
```

Comment:
```suggestion
  name: "razeedeploy-job-{{{ UUID }}}"
```

Reply:
i dont want a hanging `-` if UUID is undefined. it'll be up to the templator to prefex the uuid with a `-`